src/genie/libs/parser/iosxe/tests/ShowBgpAllNeighbors/cli/equal/golden_output7_output.txt

```diff
@@ -0,0 +1,259 @@
+#show bgp all neighbors
```

Comment:
Thank you for adding tests

Would it be possible to reduce the size of the test files but still test you changes?

Reply:
Hi Lukas, thanks for your review.
reducing the testfile size would mean reducing the test to one specific bgp neighbor config for the specific bugfix. 
Currently the test checks three different variants of bgp neighbor configs with the testfiles.
Would this make sense ?

Reply:
sorry for the late reply - didn`t notice that i have to click "resolve conversation" - I commited a shorter test for the fix

Reply:
Not a problem at all, I am replying late as well! Thank you for reducing the test file size